contributions/catalog/webapps/templates/contribute/contribute.html

```diff
@@ -592,7 +592,8 @@ <h3 id="capability">Add a Capability to this Contribution</h3>
                                 <input class="form-control envvar-value" name="environmentVariables_value_0" placeholder="Value"
                                        id="environmentVariables_value_0" type="password">
                                 <i class="bi bi-eye-slash field_icon" id="togglePassword"></i>
-                                <br>
+                                <i class="bi bi-x-lg" id="envvarclose"></i>
```

Comment:
Let's use hyphenated words for element ID names and CSS class names. So, something like `env-var-close` or `env-var-close-btn` may be better. Thanks.

Reply:
updated

Reply:
Thanks.